### PR TITLE
Modified JS File

### PR DESCRIPTION
**Pull request by myjeepxx**
He @katzzankit , I have modified JS File.

### CODE REVIEW

File: Index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
 <!DOCTYPE html>
 <html>
 <body>
-<p id="demo">he value of z is.</p>
+<p id="demox">he value of x is.</p>
```
Comment (katzzankit) [inline Index.js]: Can Use better Variables.
demoxx
demoyy
demozz

Comment (myjeepxx) [reply Index.js]: I am going to do that.

**Review by katzzankit — APPROVED** [review]
Approved.

**Comment by myjeepxx** [comment]
Merged it.

**Pull request merged**